content/r/concepts/csv-files/terms/subset/subset.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'subset()'
+Description: 'Returns a subset of an object that matches the specified conditions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Operators'
+  - 'Arithmetic'
+  - 'Comparison'
+  - 'Logical'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+The `subset()` function returns a subset of an object that matches the specified conditions.
```

Comment:
What types of object does this support?

Reply:
This supports vectors, matrices, and data frame. I'm thinking just leave it as data frames in the wording and then all these term entries can be added to a data frame entry like mentioned in your main comment.